src/components/ArticleContainer.jsx

```diff
@@ -0,0 +1,43 @@
+import { useEffect, useState } from "react";
+import { apiClient } from "../utils/apiClient";
+import { useParams } from "react-router-dom";
+import {
+  articleContainer,
+  articleHeader,
+  authorAndVotes,
+  writtenBy,
+  articleImg,
+} from "../styles/ArticleContainer.module.css";
+
+export const ArticleContainer = () => {
+  const { article_id } = useParams();
+  const [article, setArticle] = useState({});
+  const [author, setUser] = useState({});
+
```

Comment:
We need some loading feedback for a better UX - let the user know that something is happening/data is being requested 💡